tests/Library/Vanilla/Embeds/EmbedManagerTest.php

```diff
@@ -253,6 +255,34 @@ public function provideRenderedData() {
             </svg>
         </button>
     </div>
+</div>'
+            ],
+            [
+                [
+                    "url" => "https://dave.wistia.com/medias/0k5h1g1chs",
+                    "type" => "wistia",
+                    "name" => "Lenny Delivers a Video - oEmbed",
+                    "body" => null,
+                    "photoUrl" => "https://embed-ssl.wistia.com/deliveries/99f3aefb8d55eef2d16583886f610ebedd1c6734.jpg?image_crop_resized=960x540",
+                    "height" => 540,
+                    "width" => 960,
+                    "attributes" => [
+                        "thumbnail_width" => 540,
+                        "thumbnail_height" => 960,
+                        "videoID" => "0k5h1g1chs",
+                        "embedUrl" => "https://fast.wistia.net/embed/iframe/0k5h1g1chs",
+                    ]
+                ],
+                '<div class="embed-video embed embedVideo">
+    <div class="embedVideo-ratio is16by9" style="">
+        <button type="button" data-url="https://fast.wistia.net/embed/iframe/0k5h1g1chs" aria-label="Lenny Delivers a Video - oEmbed" class="embedVideo-playButton iconButton js-playVideo" style="background-image: url(https://embed-ssl.wistia.com/deliveries/99f3aefb8d55eef2d16583886f610ebedd1c6734.jpg?image_crop_resized=960x540);" title="Lenny Delivers a Video - oEmbed">
+            <svg class="embedVideo-playIcon" xmlns="http://www.w3.org/2000/svg" viewBox="-1 -1 24 24">
```

Comment:
Lets use the `svgHTML` from https://github.com/vanilla/vanilla/pull/7385/files here.

Additionally:

- Rename the variable to be a bit more specific. We may have other svgs in the future. `playButtonSVG` or similar would be better.
- Alter all the other instances of this SVG with the variable.